docker-compose.yml

```diff
@@ -13,6 +13,10 @@ services:
       - STORAGE_API_TOKEN
       - STORAGE_API_LINKING_TOKEN
       - STORAGE_API_MAINTENANCE_URL
+      - STORAGE_API_PROJECT_IDS_IN_ORGANIZATION
```

Comment:
Tyhle věci se pak budou muset pozakládat na testingu https://connection-testing.us-east-1.keboola.com/admin/auth/login/forwardTo/%252Fadmin
 a přidat do travisu https://travis-ci.com/keboola/connection/settings to je asi to co píšeš tady https://github.com/keboola/connection/pull/2253#issuecomment-535539006 ?

Reply:
a taky to je potřeba popsat do `README.md`, jak se ty hodnoty mají nastavit.

Nevím jak to teď nastavit, tak to prosím doplň do toho souboru a já to projdu znova.

Reply:
> Tyhle věci se pak budou muset pozakládat na testingu https://connection-testing.us-east-1.keboola.com/admin/auth/login/forwardTo/%252Fadmin
> a přidat do travisu https://travis-ci.com/keboola/connection/settings to je asi to co píšeš tady [keboola/connection#2253 (comment)](https://github.com/keboola/connection/pull/2253#issuecomment-535539006) ?

Ano to som myselel



> a taky to je potřeba popsat do `README.md`, jak se ty hodnoty mají nastavit.
> 
> Nevím jak to teď nastavit, tak to prosím doplň do toho souboru a já to projdu znova.

Snazil som sa nazvat tie hodnoty aby s nich bolo jasne co sa tam ma nastavit pridal som to do `README.md` este to skontrolujem zajtra a prehadzujem na teba

Reply:
Pridanie